packages/tools/kadena-cli/src/devnet/utils/simulation/coin/crosschain-transfer.ts

```diff
@@ -0,0 +1,71 @@
+import type { ChainId, ICommandResult } from '@kadena/client';
+import { createSignWithKeypair } from '@kadena/client';
+import { transferCrossChain } from '@kadena/client-utils/coin';
+
+import type { IAccount } from '../utils.js';
+import { sender00 } from '../utils.js';
+
+export async function crossChainTransfer({
+  network,
+  sender,
+  receiver,
+  amount,
+  gasPayer = sender00,
+}: {
+  network: { host: string; id: string };
+  sender: IAccount;
+  receiver: IAccount;
+  amount: number;
+  gasPayer?: IAccount;
+}): Promise<ICommandResult> {
+  // Gas Payer validations
+  if (gasPayer.chainId !== receiver.chainId && gasPayer !== sender00) {
+    console.log(
+      `Gas payer ${gasPayer.account} does not for sure have an account on the receiver chain; using sender00 as gas payer`,
+    );
+    gasPayer = sender00;
+  }
+
+  if (!gasPayer.keys.map((key) => key.secretKey)) {
+    console.log(
+      `Gas payer ${gasPayer.account} does not have a secret key; using sender00 as gas payer`,
+    );
+    gasPayer = sender00;
+  }
```

Comment:
i think these validations should not be part of the options since hes comparing the result of 2 totally we do this also in other places, and it would complicate the option logic

Reply:
Can we add `sender00` as a predefined global account?